docs/topics/proc_show-debug-logging-openshift-nodejs.adoc

```diff
@@ -0,0 +1,88 @@
+= Show Debug Logging on OpenShift
+
+Use the the `DEBUG` environment variable in your application pod in OpenShift to enable debug logging.
+
+.Prerequisites 
+* An application with debug logging.
+* The `oc` CLI client installed.
+
+
+.Procedure
+
+. Use the `oc` CLI client to log into your OpenShift instance.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc login ...
+----
+
+. Deploy your application to OpenShift.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm run openshift
+----
++
+This runs the `openshift` npm script, which wraps direct calls to xref:about-nodeshift[nodeshift].
```

Comment:
This is specified in the `package.json` file. As above, it might be good to include an excerpt here.